src/ODriveCAN.cpp

```diff
@@ -137,22 +137,19 @@ bool ODriveCAN::getPower(Get_Powers_msg_t& msg, uint16_t timeout_ms) {
 void ODriveCAN::onReceive(uint32_t id, uint8_t length, const uint8_t* data) {
 #ifdef DEBUG
     int byte_index = length - 1;
-    Serial.println("received:");
-    Serial.print("  id: 0x");
+    Serial.println(F("received:"));
+    Serial.print(F("  id: 0x"));
     Serial.println(id, HEX);
-    Serial.print("  data: 0x");
-    while (byte_index >= 0)
-        Serial.print(msg.data[byte_index--], HEX);
-    Serial.println("");
-#endif // DEBUG
-    if (node_id_ != (id >> ODriveCAN::kNodeIdShift))
-        return;
+    Serial.print(F("  data: 0x"));
+    while (byte_index >= 0) Serial.print(msg.data[byte_index--], HEX);  // msg.data is not defined??
```

Comment:
I think the intention was `msg[...]` instead of `msg.data[...]` and was overlooked during another change.